### DISCUSSION TITLE
[2021.11.11] 모의 코테 문제 풀이 공유

### DISCUSSION THREAD

**Discussion by pyunji** [pyunji/Hyundai-ITnE_Team2#10] [Algorithm]
각자 풀이한 코드를 코멘트로 남겨주세요

**Comment by pyunji** [comment]
[현대 모의코테](https://github.com/pyunji/Hyundai-ITnE_Team2/blob/pyunji/pyunji/Final-Reviews/%ED%98%84%EB%8C%80%EB%AA%A8%EC%9D%98%EC%BD%94%ED%85%8C.java)

**Comment by SeoHoeHyeong** [comment]
[현대 모의코테](https://github.com/pyunji/Hyundai-ITnE_Team2/blob/master/SHH/Final-Reviews/%ED%98%84%EB%8C%80%EB%AA%A8%EC%9D%98%EC%BD%94%ED%85%8C.java)

**Comment by hongsam123** [comment]
[현대 모의코테](https://github.com/pyunji/Hyundai-ITnE_Team2/blob/master/ES/Final-Reviews/%ED%98%84%EB%8C%80%EB%AA%A8%EC%9D%98%EC%BD%94%ED%85%8C.java)
